app/src/main/java/org/simple/clinic/patient/sync/PatientSync.kt

```diff
@@ -50,11 +50,14 @@ class PatientSync @Inject constructor(
 
   private fun toRequest(patients: List<PatientProfile>): PatientPushRequest {
     return PatientPushRequest(
-        patients.map { (patient, address, phoneNumbers) ->
+        patients.map { (patient, address, phoneNumbers, businessIds) ->
           val numberPayloads = phoneNumbers
               .map(::phoneNumberPayload)
               .let { payloads -> if (payloads.isEmpty()) null else payloads }
 
+          val businessIdPayloads = businessIds
+              .map { it.toPayload() }
```

Comment:
FYI, this lambda can also be represented as a method reference like so:

```kotlin
val businessIdPayloads = businessIds.map(BusinessId::toPayload)
```

Not necessary to change this, but I feel this looks a little cleaner.